rolling/release-el-build.yaml

```diff
@@ -0,0 +1,141 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+
+build_environment_variables:
+  RPM_BUILD_NCPUS: '1'
+
+jenkins_binary_job_priority: 80
+jenkins_binary_job_timeout: 120
+jenkins_source_job_priority: 70
+jenkins_source_job_timeout: 30
+notifications:
+  emails:
+  - scott@openrobotics.org
+  maintainers: false
+package_blacklist:
```

Comment:
Pure curiosity is this list generated by empirical observation or automation?

Reply:
Mostly empirical observation. The staging farm will be a good environment to generate a final list prior to deployment to prod.